conbench/tests/app/test_auth.py

```diff
@@ -129,3 +137,124 @@ def test_logout_unauthenticated(self, client):
         # cannot get users page after
         response = client.get("/users/", follow_redirects=True)
         self.assert_login_page(response)
+
+
+class TestLoginOIDC(_asserts.AppEndpointTest):
+    def test_login_page_shows_sso_link(self, client):
+        r = client.get("/login/", follow_redirects=True)
+        assert "Google Login" in r.text
+
+    def test_oidc_flow_against_dex(self, client):
+
+        # r = client.get("/login/", follow_redirects=True)
+        # TODO: parse this from the HTML doc
+        # initiate_flow_url = "http://127.0.0.1:5000/api/google/"
+        # follow_redirects=True)
+        r0 = client.get("/api/google/")
```

Comment:
This relates to @jonkeane's point around [here](https://github.com/conbench/conbench/pull/454#issuecomment-1325181808).